CASMTRIAGE-7215 Update prepare_site_init.md (#5298)
Cray-HPE/docs-csm: install/prepare_site_init.md

            -keystore /data/certs.jks -storepass password -noprompt
    ```

>     > **`NOTE`** If the command is executed multiple times by oversight, then console displays as below and may be ignored

Reviewer: ```suggestion
    > **`NOTE`** If the command is executed multiple times by oversight, then the console will display the following and may be ignored
```

    > to proceed further.

>     ```test

Reviewer 2: ```suggestion
    ```text
```
